tests/unit/amo/sagas/test_users.js

```diff
@@ -80,7 +80,11 @@ describe(__filename, () => {
       mockApi.verify();
     });
 
-    it('allows exceptions to be thrown', () => {
+    // Disabled because the test setup does not handle errors thrown in
+    // `fetchCurrentUserAccount()`, even though we add `.catch()` to the root
+    // task.
+    // eslint-disable-next-line jest/no-disabled-tests
+    it.skip('allows exceptions to be thrown', () => {
```

Comment:
I agree that skipping tests that fail in unrealistic ways is a good way to move this forward, but is it likely that this will be fixable if it's due to a deliberate chance in node?  I.e. should the tests just be deleted?

Reply:
I don't know yet, which is why I deferred these questions to a different issue. I have no idea how to fix these tests but we should have some sort of coverage for this part, given that it is unusual (and annoying).